app/utils/user/index.ts

```diff
@@ -339,6 +339,10 @@ export function getNotificationProps(user?: UserModel) {
         push_status: 'online',
         push_threads: 'all',
         email_threads: 'all',
+        calls_desktop_sound: 'true',
+        calls_notification_sound: 'calm',
```

Comment:
in some places I've seen the calm string with capital C, perhaps we want to use a constant to make sure the value is what we expect?

Reply:
Makes sense; replaced all strings with an enum.